pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -954,9 +954,7 @@ func (qjm *XController) getAggregatedAvailableResourcesPriority(unallocatedClust
 					continue
 				}
 
-				preemptableAWs[value.Status.SystemPriority] = append(preemptableAWs[value.Status.SystemPriority], queueJobKey)
-				preemptableAWsMap[queueJobKey] = value
-				klog.V(10).Infof("[getAggAvaiResPri] %s: Added %s to candidate preemptable job with priority %f.", time.Now().String(), value.Name, value.Status.SystemPriority)
+				addPreemptableAWs(preemptableAWs, value, queueJobKey, preemptableAWsMap)
```

Comment:
Isn't this call already covered by line 970? Why is it here?

Reply:
Two calls are needed, one for dispatcher mode and one for agent mode.